src/statscollector/DefaultStatsCollector.ts

```diff
@@ -414,11 +414,7 @@ export default class DefaultStatsCollector implements StatsCollector {
   }
 
   isValidRawMetricReport(rawMetricReport: RawMetricReport): boolean {
-    return (
-      (this.isValidChromeRawMetric(rawMetricReport) ||
-        this.isValidStandardRawMetric(rawMetricReport)) &&
-      this.isValidSsrc(rawMetricReport)
-    );
+    return this.isValidStandardRawMetric(rawMetricReport) && this.isValidSsrc(rawMetricReport);
```

Comment:
Your PR doesnt say it - but have you tested against all the browsres and ensure they are correctly populated? Could you also update the PR on the testing steps?